wpical/src/main/native/cpp/WPIcal.cpp

```diff
@@ -58,6 +60,94 @@ void drawCheck() {
   ImGui::NewLine();
 }
 
+void processFileSelector(std::unique_ptr<pfd::open_file>& selector,
+                         std::string& selected_file) {
+  if (selector) {
```

Comment:
All of these need to also check for `selector->ready(0)` (e.g. `if (selector && selector->ready(0))`.